src/assessments/common/instructions-and-labels-note.tsx

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import * as React from 'react';
+
+import * as Markup from '../markup';
+
+export const InstructionsAndLabelsNotes = () => (
+    <Markup.Emphasis>
+        Note: For WCAG 2.1 compliance both visible instructions and labels must be programatically related to a widget. For WCAG 2.0
+        compliance, only visible instructions must be programatically related to a widget.
```

Comment:
Spelling here too. I'm getting red underline for this word in the comment editor but according to [Merriam-Webster dictionary](https://www.merriam-webster.com/dictionary/programmatic#other-words), this is how it's spelled.
```suggestion
        compliance, only visible instructions must be programmatically related to a widget.
```